### PR TITLE
Instance and ICMP removal

### PR DESCRIPTION
**Pull request by aaronabf**
We currently do not use Instances, nor ICMP which rely solely on Instances, thus I removed the views and options to create them. If for some reason we ever need these features again, we can just revert this commit, but let's not give users unnecessary buttons and things to mess up. In no way will this change any data operations, just hides the options.

### CODE REVIEW

**Comment by mahendra** [comment]
also please do check on prod cabot that we are not using ICMP / Instances.

File: cabot/cabotapp/models.py [hunk 1]
```diff
@@ -542,7 +536,8 @@ def description(self):
             return 'ICMP Reply from {}'.format(self.instance_set.all()[0].address)
         return 'ICMP Reply'
 
-    update_url = 'update-icmp-check'
+    # Unless we unlink this status from everywhere, we just need to provide an alternative url.
+    update_url = 'update-jenkins-check'
```
Comment (elainearbaugh) [inline cabot/cabotapp/models.py]: Let's just take them back to `check` (the page listing check attributes).

File: cabot/templates/cabotapp/statuscheck_detail.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
       {% if check.polymorphic_ctype.model == 'jenkinsstatuscheck' %}
         <a href="{% jenkins_human_url check.name %}" class=""><i class="glyphicon glyphicon-link"></i></a>
       {% endif %}
-      <a href="{% url check.update_url pk=check.id%}"><i class="glyphicon glyphicon-edit"></i>
+      <a href="{% url check.update_url pk=check.id%}"><i class="glyphicon glyphicon-edit"></i></a>
```
Comment (elainearbaugh) [inline cabot/templates/cabotapp/statuscheck_detail.html]: 👍  :(

**Comment by elainearbaugh** [comment]
The only icmp check/instance on prod is the one IT made for hipchat.team.affirm.com, which is disabled (since the ping check wasn't working). So we should be good to remove it

File: cabot/templates/cabotapp/statuscheck_detail.html (hunk 2 above)
Comment (mahendra) [inline cabot/templates/cabotapp/statuscheck_detail.html]: 👍

File: cabot/cabotapp/models.py (hunk 1 above)
Comment (aaronabf) [inline cabot/cabotapp/models.py]: Regardless, this entirely block of code will be removed in the TCP commit, so should I even bother with any changes. And on that note, should we wait to do extensive QA until a everything is in?

**Comment by aaronabf** [comment]
Created new PR with more complete removal on new branch.

**Pull request closed without merging**